README.md

```diff
@@ -1,4 +1,4 @@
-# neurobot - A Matrix Workflow Builder
+# ![neurobot logo](https://github.com/Automattic/neurobot/blob/master/resources/icon.svg?raw=true) neurobot - A Matrix Workflow Builder
```

Comment:
I noticed that the icon doesn't show when viewing the [rendered file](https://github.com/Automattic/neurobot/blob/5dda2e5ed4fd5d8a61fca79170bd46f96af39c16/README.md). I'm guessing the intention was that the icon would render?

Reply:
Yeah! But if it renders here, it won't render without a change in master branch, so have the master branch url here. Wish we could specify a branch variable placeholder in urls.

Reply:
Ah gotcha, I didn't notice the `master` in the URL, makes sense.